HW_3/HW03.Alphabet/HW03.Alphabet/bin/Debug/net5.0/HW03.Alphabet.deps.json

```diff
@@ -0,0 +1,23 @@
+{
+  "runtimeTarget": {
+    "name": ".NETCoreApp,Version=v5.0",
+    "signature": ""
+  },
+  "compilationOptions": {},
+  "targets": {
+    ".NETCoreApp,Version=v5.0": {
+      "HW03.Alphabet/1.0.0": {
+        "runtime": {
+          "HW03.Alphabet.dll": {}
+        }
+      }
+    }
+  },
+  "libraries": {
+    "HW03.Alphabet/1.0.0": {
+      "type": "project",
+      "serviceable": false,
+      "sha512": ""
+    }
+  }
+}
```

Comment:
many files - should be in .gitignore
![image](https://user-images.githubusercontent.com/3023032/124420080-763cd400-dd67-11eb-8eb9-184effc01fb0.png)